index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html><html lang="en">
+<head>
+<link rel="stylesheet" href="assets/style.css" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+<body>
+    <header><img src="assets/logo.png" alt="YouTube logo" /></header>
+    <main>
+        <section class="content-area">
+            <div class="video-block">
+                <div class="video-embed">
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/DK7Urjt7vr8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                </div>
+                <div class="video-title">RISING SHADOWS EVENT TIPS & TRICKS | POKÉMON GO</div>
+            </div>
+        </section>
+        <section class="sidebar">
```

Comment:
- It may be better to add the sidebar as an aside instead of a div inside main

Reply:
> * It may be better to add the sidebar as an aside instead of a div inside main

- [ ] Always includes an explanation, a "why", for each requested change. For example
- This is a well-explained request: I would kindly suggest you to add the sidebar as an aside instead of a div inside main to improves the clarity of HTML documents.